### PR TITLE
chore: prepare for release of v0.1.0

### PR DESCRIPTION
**Pull request by jopit**
Signed-off-by: John Pitman <jpitman@redhat.com>

**What type of PR is this?**

> /kind enhancement

**What does this PR do / why we need it**:

Adds/Updates the artifacts needed to release v0.1.0

### CODE REVIEW

File: config/manifests/bases/argocd-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -660,4 +660,5 @@ spec:
   maturity: alpha
   provider:
     name: Argo CD Community
+  replaces: argocd-operator.v0.0.15
   version: 0.0.0
```
Comment (wtam2018) [inline config/manifests/bases/argocd-operator.clusterserviceversion.yaml:664]: verison 0.1.0, right?

File: deploy/olm-catalog/argocd-operator/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+# Use `replaces-mode` or `semver-mode`. Once you switch to `semver-mode`, there is no easy way back.
+updateGraph: replaces-mode
+reviewers:
+  - jomkz 
+  - jannfis
+  - wtam2018
```
Comment (wtam2018) [inline deploy/olm-catalog/argocd-operator/ci.yaml:7]: please add yourself and iam-veeramalla

File: config/manifests/bases/argocd-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -660,4 +660,5 @@ spec:
   maturity: alpha
   provider:
     name: Argo CD Community
+  replaces: argocd-operator.v0.0.15
```
Comment (wtam2018) [inline config/manifests/bases/argocd-operator.clusterserviceversion.yaml:663]: Did you have to edit the generated CSV to add the correct version (i.e. 0.1.0)?  If so, you can also add the "replaces" field as well and the build/release process have one less file (this file) to touch.

Comment (jopit) [reply config/manifests/bases/argocd-operator.clusterserviceversion.yaml:663]: No, the correct version was automatically added to the generated CSV.

File: deploy/olm-catalog/argocd-operator/ci.yaml (hunk 2 above)
Comment (jopit) [reply deploy/olm-catalog/argocd-operator/ci.yaml:9]: Done

File: hack/env.sh [hunk 4]
```diff
@@ -49,15 +49,15 @@ export ARGOCD_OPERATOR_REGISTRY_BUILD_DIR=${ARGOCD_OPERATOR_REGISTRY_BUILD_DIR:-
...
 export ARGOCD_OPERATOR_REGISTRY_IMAGE=${ARGOCD_OPERATOR_REGISTRY_IMAGE:-"${ARGOCD_OPERATOR_REGISTRY_IMAGE_REPO}:${ARGOCD_OPERATOR_REGISTRY_IMAGE_TAG}"}
 
 # Operator util vars
 export ARGOCD_OPERATOR_UTIL_BUILD_DIR=${ARGOCD_OPERATOR_UTIL_BUILD_DIR:-"build/util"}
 export ARGOCD_OPERATOR_UTIL_IMAGE_HOST_ORG=${ARGOCD_OPERATOR_UTIL_IMAGE_HOST_ORG:-${ARGOCD_OPERATOR_IMAGE_HOST_ORG}}
 export ARGOCD_OPERATOR_UTIL_IMAGE_NAME=${ARGOCD_OPERATOR_UTIL_IMAGE_NAME:-"${ARGOCD_OPERATOR_NAME}-util"}
-export ARGOCD_OPERATOR_UTIL_IMAGE_REPO=${ARGOCD_OPERATOR_UTIL_IMAGE_REPO:-"quay.io/jmckind/${ARGOCD_OPERATOR_UTIL_IMAGE_NAME}"}
+export ARGOCD_OPERATOR_UTIL_IMAGE_REPO=${ARGOCD_OPERATOR_UTIL_IMAGE_REPO:-"${ARGOCD_OPERATOR_IMAGE_HOST_ORG}/${ARGOCD_OPERATOR_UTIL_IMAGE_NAME}"}
 export ARGOCD_OPERATOR_UTIL_IMAGE_TAG=${ARGOCD_OPERATOR_UTIL_IMAGE_TAG:-${ARGOCD_OPERATOR_BRANCH_NAME}}
 export ARGOCD_OPERATOR_UTIL_IMAGE=${ARGOCD_OPERATOR_UTIL_IMAGE:-"${ARGOCD_OPERATOR_UTIL_IMAGE_REPO}:${ARGOCD_OPERATOR_UTIL_IMAGE_TAG}"}
```
Comment (wtam2018) [inline hack/env.sh:63]: I thought we were no longer using this shell.  Please convert it to a target in Makefile.

File: version/version.go [hunk 5]
```diff
@@ -16,5 +16,5 @@ package version
 
 var (
 	// Version is the ArgoCD Operator version.
-	Version = "0.0.16"
+	Version = "0.1.0"
```
Comment (wtam2018) [inline version/version.go:19]: Please get rid of this version file/package.   Instead, we can set the -X in -ldflags and use VERSION variable in Makefile.  Please follow the pattern e.g. in kam.
https://github.com/redhat-developer/kam/blob/master/Makefile#L10
to set the Version variable in go code.
https://github.com/redhat-developer/kam/blob/master/pkg/cmd/version/version.go#L13

Comment (jopit) [reply version/version.go]: Done

File: hack/env.sh (hunk 4 above)
Comment (jopit) [reply hack/env.sh]: I've cleaned up the scripts folder.

**Review by wtam2018 — APPROVED** [review]
LGTM
Thanks @jopit !

**Review by wtam2018 — APPROVED** [review]
LGTM

**Pull request merged**